README.md

```diff
@@ -2,16 +2,33 @@
 
 [![Build Status](https://travis-ci.org/sonatype-nexus-community/nexus-repository-apt.svg?branch=master)](https://travis-ci.org/sonatype-nexus-community/nexus-repository-apt) [![DepShield Badge](https://depshield.sonatype.org/badges/sonatype-nexus-community/nexus-repository-apt/depshield.svg)](https://depshield.github.io)
 
+> **WARNING: 3.17.0 includes a feature flagged version of the APT plugin by default**. We're releasing this way so that 
+we can get early feedback on the APT plugin without impacting the wider user base. From 3.18.0 APT will be 
+enabled by default.
+
+>If you are an APT user upgrading to 3.17 setting ```nexus.apt.enabled=true``` in ```./etc/nexus.properties``` will 
+enable the plugin. **You will not be able to install the community plugin.**
+
+>There are some differences from the community version of the plugin. First of all, we have added component records for 
+the Debian files which adds support for features such as *Search*, *Cleanup Policies*, *Tagging (PRO only)*
+and *Moving between repositories (PRO only)*. We've also removed the "retain N versions" feature as this conflicts with 
+Cleanup Policies and future work we are doing in that space.
+
+> We've added support for other core features: *Restore Metadata Task*, *Routing Rules*, *API for Repository creation
+via Groovy*, *Improved tree browsing* and we've added *help documentation (released with 3.18)*.
```

Comment:
Trivial suggestion: "(**to be** released with 3.18)" (assuming you keep 3.18 notes).  Otherwise another strike point.

Reply:
Q: How is it "Improved"?  Is it just "tree browsing" with your favorite adjective?=)

Reply:
https://github.com/sonatype-nexus-community/nexus-repository-apt/pull/116/commits/b9ba7f103fd77fb5dc4f785eecddc6e1cd1dc0fe

I'd thought we added in the name and version folders for packages but actually what we did was add them *back* in after breaking it when adding component support :).